pkgs/applications/video/makemkv/default.nix

```diff
@@ -1,34 +1,48 @@
-{ stdenv, fetchurl
-, openssl, qt5, libGLU_combined, zlib, pkgconfig, libav
```

Comment:
Is libav still maintained? I thought everyone had moved back to ffmpeg.

Reply:
The MakeMKV page recommends a recent ffmpeg, so I have changed the dependency to ffmpeg as you suggested. Also seems to work well.